cogs5e/models/sheet/base.py

```diff
@@ -28,27 +28,40 @@ def to_dict(self):
     @classmethod
     def default(cls):
         return cls(0, 10, 10, 10, 10, 10, 10)
+    
+    @classmethod
+    def custstats(cls,cust,origstats):
+        stats=[]
+        abils=('pro',)+STAT_ABBREVIATIONS
+        for i in range(len(abils)):
+            stat=abils[i]
+            stats.append(cust[stat] if stat in cust.keys() else origstats.get_mod(stat))
+        return cls(stats[0],stats[1],stats[2],stats[3],stats[4],stats[5],stats[6])
+            
 
     def get_mod(self, stat: str):
         abbr_stat = stat.lower()[:3]
-        if abbr_stat not in STAT_ABBREVIATIONS:
+        abils=('pro',)+STAT_ABBREVIATIONS
+        if abbr_stat not in abils:
             raise ValueError(f"{stat} is not a valid stat.")
         return {
-            'str': self.strength // 2 - 5, 'dex': self.dexterity // 2 - 5,
+            'pro': self.prof_bonus, 'str': self.strength // 2 - 5, 'dex': self.dexterity // 2 - 5,
             'con': self.constitution // 2 - 5, 'int': self.intelligence // 2 - 5,
             'wis': self.wisdom // 2 - 5, 'cha': self.charisma // 2 - 5
         }[abbr_stat]
 
     def __str__(self):
-        return f"**STR**: {self.strength} ({(self.strength - 10) // 2:+}) " \
+        return f"**PRO**: +{self.prof_bonus} " \
```

Comment:
```suggestion
        return f"**PRO**: {self.prof_bonus:+} " \
```
This also breaks the display of the character sheet by causing proficiency bonus to appear twice.